databricks/koalas/missing/indexes.py

```diff
@@ -73,7 +73,6 @@ class _MissingPandasLikeIndex(object):
     is_type_compatible = unsupported_function('is_type_compatible')
     join = unsupported_function('join')
     map = unsupported_function('map')
-    nunique = unsupported_function('nunique')
```

Comment:
could you add `MultiIndex.nunique` also and just raise `NotImplementedError` for `MultiIndex` like pandas?

```python
>>> pidx = pd.MultiIndex.from_tuples([('a', 1), ('b', 2)])
>>> pidx.nunique()
Traceback (most recent call last):
...
NotImplementedError: isna is not defined for MultiIndex
```